lib/msal-common/src/error/ClientConfigurationError.ts

```diff
@@ -47,7 +47,7 @@ export const ClientConfigurationErrorMessage = {
     },
     invalidPrompt: {
         code: "invalid_prompt_value",
-        desc: "Supported prompt values are 'login', 'select_account', 'consent', 'create' and 'none'.  Please see here for valid configuration options: https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#commonauthorizationurlrequest",
+        desc: "Supported prompt values are 'login', 'select_account', 'consent', 'create', 'none' and 'no_session'.  Please see here for valid configuration options: https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#commonauthorizationurlrequest",
```

Comment:
Is the included link updated to include "no_session"?

Reply:
Doesn't look like it: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/0a0ca9b006f775309090fafa33fde6a9aae550ad/lib/msal-common/src/request/CommonAuthorizationUrlRequest.ts#L24-L29

Reply:
This link points to the typedocs - which will be updated automatically when this gets released

Reply:
It may also be good to confirm if no_session is only meant to be used internally. There are no public docs for it.

Reply:
Fine documenting it in typedocs regardless.